manifests/04_oauthserver_trust_role.yaml

```diff
@@ -0,0 +1,17 @@
+apiVersion: rbac.authorization.k8s.io/v1
+kind: Role
+metadata:
+  annotations:
+    rbac.authorization.kubernetes.io/autoupdate: "true"
```

Comment:
This should not be needed. We should not honor a "false" here, so it doesn't matter.